.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav_v2.vtl

```diff
@@ -381,11 +384,15 @@
     <ul>
       ## First list item is always Overview
       <li>
-        <a class="icon $overviewIconClass" href="$overviewLink">Overview</a>
```

Comment:
@kaylaziegler did some final review of the sample pages using this updated format and found that the 2 external links in the global nav dropdowns that are for athletics and events calendar were appending the site:// to the link when they shouldn't. I added an IF clause to test. Seems to work fine now as seen at https://dev-www.chapman.edu/pharmacy/support.aspx. Soooo, can you review these lines are 387+ and re-approve?

Reply:
@kaylaziegler omg. good catch. I edited the file instead of copying the whole file I'd worked out on dev-cascade. thank you so much. glad we have code reviews.